build-settings/create-babel-plugins.js

```diff
@@ -1,4 +1,4 @@
 /* eslint-disable import/no-commonjs */
```

Comment:
We should probably just remove this file, right?

Reply:
I left it here in case we want to add plugins in the future.  @somewhatabstract how do you feel about us getting rid of this file?

Reply:
If it's not used, we don't need it.

Reply:
This file gets removed in a later PR.